src/atoms/Space/Space.native.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
```

Comment:
`import styled from 'styled-components/native';`

Reply:
Should we add a linter warning for this? :/

Reply:
Yes